README.md

```diff
@@ -108,27 +108,29 @@ async def main(req: func.HttpRequest, context: func.Context) -> func.HttpRespons
 
 ### Testing locally
 
-First run the command below to install the necessary requirements.
+1. Create a [Python virtual environment](https://docs.python.org/3/tutorial/venv.html#creating-virtual-environments) and activate it.
 
-```log
```

Comment:
Most of these changes are just indentation, to align blocks under each numbered steps.